docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -25,6 +25,17 @@ Yes. Customers can subscribe to Teleport Enterprise Cloud updates at [status.tel
 
 Please [reach out to sales](https://goteleport.com/signup/enterprise) to discuss pricing.
 
+## Are maintenance update times configurable? 
+
+Yes, Teleport provides scheduled maintenance windows for upgrades, patches and other required updates. You can select the
+maintenance start window time of day that works best for your organization.
+
+To select the best maintenance time window select your username in the Web UI, then the Help & Support option from the drop-down.
+The Schedule Upgrade section within that page allows you to configure the start time. Updates for that tenant will not start 
```

Comment:
```suggestion
The Scheduled Upgrade section within that page allows you to configure the start time. Updates for that tenant will not start 
```

Is it schedule or scheduled? I thought it was scheduled.

Reply:
Thanks I've matched it @zmb3

Reply:
![image](https://user-images.githubusercontent.com/60704961/232163037-1f4e12d7-349f-4928-bd2c-4bd8605e953e.jpeg)